app/models/project.rb

```diff
@@ -12,6 +12,11 @@ class Project < ApplicationRecord
 
   scope :active, -> { where.not(status: "archived").or(where(status: nil)) }
   scope :parents, -> { where(parent: nil) }
+  scope :with_ordered_stories, ->(project_id) {
```

Comment:
maybe the name of the scope is a bit confusing? the name is `with_ordered_stories` but it's ordering the projects and I have the feeling that it's not actually ordering the stories inside each project